### ISSUE TITLE
Mobile support

### ISSUE THREAD

**Issue by nwisehart** [Esri/a11y-map#6]
@jimmieego : Jun 14
We need to check if the demo app and its pattern work on mobile devices. I will report any findings here.
***
@jimmieego : Jul 19
Tested VoiceOver in iOS with the prototype and the web version of Google Map:

- VoiceOver uses the swipe left/right to move focus forward/backward, which is equivalent to Tab/Shift+Tab on desktop. So I cannot use one finger to drag and pan the map.
- With VoiceOver running, I cannot set focus to the map.
- With VoiceOver running, I cannot use two finger pinch gesture to zoom in/out the map (note this is different from zooming in/out of the whole screen). I can set focus to the zoom controls in the prototype and then zoom in/out the map, but there is no feedback of the current zoom level.

With the zooming and panning, a workaround is to temporarily disable VoiceOver and then do the normal zooming or panning, and enable VoiceOver again.

In Settings > General > Accessibility, scroll all the way down to Accessibility Shortcut, turn on the Switch Control and VoiceOver. This will allow you to triple click the Home button to show a quick menu to toggle VoiceOver on or off.
***
@jimmieego : Jul 19
Also a short video on how to use VoiceOver in iOS: https://youtu.be/bCHpdjvxBws
***
See original at: https://github.com/kellyhutchins/a11y-map/issues/13
